packages/pharos/src/components/button/pharos-button.ts

```diff
@@ -112,12 +124,41 @@ export class PharosButton extends ScopedRegistryMixin(FocusMixin(AnchorElement))
   public large = false;
 
   /**
+   * @deprecated
```

Comment:
I may be incorrect, but does Pharos v14 introduce the new accessibility properties on most components? I guess I ask because if we released this with v14 we could remove the "deprecated" attributes entirely instead of providing backwards compatibility. I imagine at some point or another we would want to remove those attributes which would be a breaking change and a major version bump.

Reply:
Hey @shoupeva-ithaka, for the v14 release, we'll be removing the backwards compatibility entirely so the deprecated callouts will also be removed. This PR is for the minor release to get those folks who are ready to move to the new attributes a way to do so, while flagging for other folks that it'll be deprecated come the v14 major release.

Reply:
Also, to follow up, when v14 is released it won't be adding the attributes to the components per se, consumers will still need to update the attributes on the current implementation of specific components.